Makefile

```diff
@@ -331,3 +331,37 @@ echo "Downloading $(2)" ;\
 GOBIN=$(TOOLSDIR) go install $(2) ;\
 }
 endef
+
+# dev environment
```

Comment:
I added all of this to the bottom of the Makefile to avoid annoying conflicts with #755. Will reorder once this is in better shape